docs/developer/releases.md

```diff
@@ -0,0 +1,33 @@
+# Kubeapps Releases Developer Guide
+
+The purpose of this document is to guide you through the process of releasing a new version of Kubeapps.
+
+## 1 - Create a new git tag
+
+The first step is to tag the repository master branch tip and push it upstream. It is important to note that the tag name will be used as release name.
+
+```bash
+export VERSION_NAME="v1.0.0-beta.1"
+
+git tag ${VERSION_NAME}
+# upstream remote being kubeapps/kubeapps
+git push upstream ${VERSION_NAME}
```

Comment:
this might be confusing, let's use `origin` as that is more common, or omit the remote name and assume that the default is correct.

Reply:
agree, I'm using `origin`: the command I usually execute is `git push --tags`

Reply:
I put it this way because I thought that we wanted to encourage people to work with a fork. In which case it is common to have `origin` as your fork but `upstream` as the forked repo. That way pushing to `upstream` is a conscious decision. 

I can change it though.

```
git push --tags
```

I wanted to avoid this because it can push tags that you might have locally and did not want to push. My proposal seems safer?

Reply:
I think it's fine, this doc is just for us and we understand what that means.